package.json

```diff
@@ -42,10 +45,11 @@
     "mocha": "^10.0.0",
     "nock": "^13.1.3",
     "nyc": "^15.1.0",
-    "pem": "^1.14.4",
+    "pem": "Dexus/pem#master",
```

Comment:
Could not we use https://www.npmjs.com/package/self-cert ?

Reply:
Thought about it. I will just generate the rsa keys once. Then we dont need pem at all.

Reply:
See comment below ;)